rotkehlchen/globaldb/handler.py

```diff
@@ -1305,31 +1322,31 @@ def get_all_manual_latest_prices(
             return read_cursor.fetchall()
 
     @staticmethod
-    def delete_manual_latest_price(asset: Asset) -> Asset:
+    def delete_manual_latest_price(asset: Asset) -> List[Tuple[Asset, Asset]]:
         """
-        Deletes manual current price from globaldb and returns the asset that was being used as
-        reference for the price
+        Deletes manual current price from globaldb and returns the list of asset
+        price pairs to be invalidated.
         May raise:
         - InputError if asset was not found in the price_history table
         """
         with GlobalDBHandler().conn.write_ctx() as write_cursor:
-            # check the asset of reference that will be deleted
+            # get asset pairs to invalidate
```

Comment:
it would be nice if you added here a comment explaining why we get multiple pairs here